### PR TITLE
Introduce #readFromContext: for all variables

### PR DESCRIPTION
**Pull request by dionisiydk**
- introduces #readInContext: for all variables and refactors #lookupSymbol:.
As result it fixes #lookupSymbol: to work with all kind of slots (it was only working for simple inst vars).
- introduce write:inContext for all variables.

Having pair of those methods we can get read and write any kind of variable having context instance.

### CODE REVIEW

File: src/Debugging-Core/Context.extension.st [hunk 1]
```diff
@@ -22,15 +22,13 @@ Context >> callPrimitive: primNumber [
 { #category : #'*Debugging-Core' }
 Context >> lookupSymbol: aSymbol [
```
Comment (dionisiydk) [inline src/Debugging-Core/Context.extension.st:23]: I think we should rename this method to something like #valueOfSymbol:. I can do it in another PR. Suggestions for name are welcome.

Comment (MarcusDenker) [reply src/Debugging-Core/Context.extension.st:23]: yes, the name is not good... maybe valueForVariableNamed? valueForSymbol:?

**Review by MarcusDenker — APPROVED** [review]
Good! Sadly the CI is down...

File: src/Debugging-Core/Context.extension.st (hunk 1 above)
Comment (dionisiydk) [reply src/Debugging-Core/Context.extension.st:23]: and we need another name to modify the value. Basically they will be generalized version of tempNamed:/tempNamed:put:

Comment (dionisiydk) [reply src/Debugging-Core/Context.extension.st:23]: I think #read/#write are good words to use there:
- aContext readVarNamed: aSymbol
- aContext write: aValue toVarNamed: aSymbol

**Comment by MarcusDenker** [comment]
testReifyAssignmentTempOperationBefore is failing

**Comment by MarcusDenker** [comment]
the reason is that OCTempVariable #write:inContext:  misses a return at the end.

(writing a variable used to return the value.. no idea if that is bad or good)

**Pull request merged**